core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -671,8 +671,10 @@ static void map_memarea(struct tee_mmap_region *mm, uint32_t *ttb)
 	 * TODO: support mapping devices at a virtual address which isn't
 	 * the same as the physical address.
 	 */
-	if (mm->va < (NUM_UL1_ENTRIES * SECTION_SIZE))
-		panic("va conflicts with user ta address");
+	if (ttb == (void *)core_mmu_get_main_ttb_va()) {
```

Comment:
@etienne-lms I am building a mmu table in IRAM for suspend/resume usage. There is no userspace mapping, the `mm->va` maybe inside the `0-64M` space, I do not want that panic.